packages/templates/clients/websocket/python/example.py

```
@@ -1,5 +1,5 @@
-from temp.snapshotTestResult.client_postman import PostmanEchoWebSocketClientClient
-import time
+from temp.snapshotTestResult.client_hoppscotch import HoppscotchEchoWebSocketClient
```

Ruff (0.8.2)

`temp.snapshotTestResult.client_hoppscotch.HoppscotchEchoWebSocketClient` imported but unused

Remove unused import: `temp.snapshotTestResult.client_hoppscotch.HoppscotchEchoWebSocketClient`

(F401)